resources/assets/sass/libs/_library-variable-overrides.scss

```diff
@@ -19,7 +19,7 @@ $brand-danger: #bf5329;
 // Typography
 $icon-font-path: "~bootstrap-sass/assets/fonts/bootstrap/";
```

Comment:
This is probably not accurate anymore, can you add here all variables from bootstrap 4?
Sot that we don't need to transfer them to each project.

Reply:
Yeah, sure :)